Hide congrats confetti when not on inbox tab

Francisca105 opened the pull request:
### Celebration Component Improvements

#### Component Updates:
- `apps/web/components/Celebration.tsx`:  
  - Added optional `confetti` prop to control the display of confetti.  
  - Conditionally render the "Congrats!" text and confetti based on the `confetti` prop.

#### Integration:
- `apps/web/components/email-list.tsx`:  
  - Updated to pass the `confetti` prop dynamically (`type !== "inbox"`) when using the `Celebration` component.

Closes #123 

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Enhancements**
  - Updated the empty email list display: now shows a celebration message only for the inbox, while other folders display a neutral "No emails to display" message.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/elie222/inbox-zero?pullRequest=390) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/elie222/inbox-zero?pullRequest=390) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/elie222/inbox-zero?pullRequest=390) it.</sub>

Francisca105 commented:
@elie222 Could you please take a look? Would love to hear your thoughts!

Francisca105 commented:
I've updated the code to display a simple message instead of using the ``Celebration`` component outside of the inbox, as requested. Here's a preview of how it looks now on the page:

![image](https://github.com/user-attachments/assets/e61fdf77-c4e6-4bdf-9e38-3aae2e8bccb0)

I also added some basic styling to maintain consistency (you can check out the changes [here](https://github.com/elie222/inbox-zero/pull/390/files#diff-8a440eb2743cdcc3281780e0489d1805dd0d003196f5b017034acb35d64af5c0R151))

Let me know if this looks good to you or if you'd prefer any adjustments!

Francisca105 commented:
@elie222 Could you take a look at the changes and let me know if this meets what you had in mind? Thanks!
